server/events/project_command_pool_executor.go

```diff
@@ -17,7 +17,7 @@ func runProjectCmdsParallel(
 	var results []models.ProjectResult
 	mux := &sync.Mutex{}
 
-	wg := sizedwaitgroup.New(15)
+	wg := sizedwaitgroup.New(poolSize)
```

Comment:
do we need to pull in this configuration somewhere to make sure we're creating a big enough pool?

Reply:
We have this set to 15 right now through a config flag that's pass in as a env var from chamber. For some reason they hard coded this to 15 upstream, even though they support a config flag to change it 😖  - I think it was an error

Reply:
I can open a PR upstream to fix this later